app/signup/page.tsx

```diff
@@ -77,6 +78,7 @@ export default function SignUpPage() {
   };
   return (
     <div className="signup">
+      <ThemeSwitch positionFix={true} />

```

Comment:
To się powiela w wielu miejscach, może warto zrobić jakiś globalny wrapper stron, który będzie opakowywał treść podstron i on będzie miał takie rzeczy właśnie jak ten ThemeSwitch?